addons/web_editor/static/src/js/editor/image_processing.js

```diff
@@ -502,7 +502,29 @@ export async function loadImageInfo(img, recordInfo, attachmentSrc = "") {
     // "local" image even if the domain name is now "mycompany.be".
     if (imageData) {
         weUtils.updateImageDataRegistry(imgSrc, weUtils.convertSnakeToCamelOrCamelToSnakeObject(imageData, true));
+        return;
     }
+    // No image.data or original attachment has been found (e.g; image field
+    // where the image comes from the backend and that never has been modified).
+    await fetch(src).then((result) => {
```

Comment:
- Feels a test to cover that case could be nice :pray: 
- More generically: make sure to check if you could improve any test about this whole PR. We already have tours that goes through the editor to test image options -> would be nice that after the tour, the python part of the test also checks that the right records were created/updated :pray: